### DISCUSSION TITLE
How to allow fine-grained API token access to images uploaded in pull request comments?

### DISCUSSION THREAD

**Discussion by tino** [community/community#69034] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Images added to PR bodies and comments are accessible on an url like `https://github.com/ORG/REPO/assets/37123687/6728aaf6-9705-4c59-90f2-6769d13ca331`. 

When I call that URL with a classic token, it returns a 301 response with a `Location: https://github-production-user-asset-6210df.s3.amazonaws.com/37123687/6728aaf6-9705-4c59-90f2-6769d13ca331.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=..........`

However even with all permissions set to read on a fine-grained token, the same request with such token will return a 404 Not Found.

Is it possible to access these assets with those new tokens? 

PS. It is quite unclear what to call these assets. There is no mention of them in any docs as far as I could find.

**Comment by YmerejRedienhcs** [comment]
I have the same request with respect to the images linked in issues and comments, but with the classic access tokens, since the fine-grained tokens don't work with repos that I don't own.  My use case is that I want to copy/reproduce issues (typically bugs) from one repo to another, possibly unrelated, repo.  For example, from a GitHub repo to a GitHub Enterprises repo.  I want to reproduce the [images that are attached to the original issues] in the new issues.

Basically, I want the API to support downloading images from issue body and comments, and the ability to supply new images and link them into new/existing issue bodies and comments, the same capability as is supported from the browser.
